src/ui_ng/lib/src/repository-stackview/repository-stackview.component.ts

```diff
@@ -153,15 +168,64 @@ export class RepositoryStackviewComponent implements OnInit {
     this.clrLoad(st);
   }
 
+  saveSignatures(event: {[key: string]: string[]}): void {
+    Object.assign(this.signedCon, event);
```

Comment:
Let event={repoName:this.repoName, signedNames:[]};
...

this.signedCon[event.repoName]=event.signedNames;